src/JestExt.ts

```diff
@@ -412,16 +413,25 @@ export class JestExt {
     this.channel.appendLine(noANSI)
   }
 
+  private handleJestEditorSupportEvent(output: string) {
+    if (output.includes('onRunStart')) {
+      this.status.running('Running tests')

```

Comment:
now we have a precise knowledge of jest starts, I think there is a great opportunity to consider killing the `clearOnNextInput`, which at times cleared the output channel prematurely. Do you think we can optimize and consolidate that logic based on 'onRunStart' instead?

Reply:
Good insight. I removed `clearOnNextInput` as it looks to be not needed anymore.